src/Parsing/Common/SummaryHelper.php

```diff
@@ -49,13 +73,14 @@ public static function formatForDisplay($inputString = null, ?int $maxColSize =
         if ($inputString === false) {
             return 'False';
         }
-        if (!$inputString || strlen($inputString) == 0) {
+        $inputString = SummaryHelper::escapeSpecialChars($inputString);
+        if (!$inputString || mb_strlen($inputString) == 0) {
             return "";
         }
         if (!isset($maxColSize)) {
             return $inputString;
         }
-        return strlen($inputString) <= $maxColSize ? $inputString : substr(
+        return mb_strlen($inputString) <= $maxColSize ? $inputString : mb_substr(
```

Comment:
1. Ternary expressions are harder to read, prefer more explicit if/else, even if longer.
2. Set encoding explicitly on mb_xxx functions.

Reply:
I KNEW I was going to miss one :angry:

Reply:
See point n° 1 ;-)